.github/workflows/release-update-xdoc-with-releasenotes.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Push Release Notes.
```

Comment:
Same.

Reply:
Push release notes to releasenotes.xml file

Reply:
Done.